apps/poker-roll/src/app/bankrollPage.jsx

```diff
@@ -1,13 +1,35 @@
 import React from "react";
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
 
 
+function PastSession(props){
+    return (
+        <div>
+            <Container maxWidth="sm">
+                <Box>
+                    <h4>Date</h4>
+                    <p>GAME Played</p>
+                    <span>+- result</span>
+                </Box>
+            {/* show info of previous sessions */}
+            {/* date/ location/ game type/ plus/minus of session */}
```

Comment:
I'd just start basic with the previous session list, can come back later and add in the expand functionality to show more details.